.github/README_fr.md

```diff
@@ -58,29 +58,29 @@
 Goxygen vise à vous faire gagner du temps lors de la mise en place d'un nouveau projet. Il
 crée un squelette d'application avec une configuration complète par défaut.
 Vous pouvez commencer immédiatement à implémenter votre logique métier.
-Goxygen génère du code Backend en Go, le connecte aux composants Frontend, fournit un 
-Dockerfile pour l'application et crée des fichiers Docker-compose pour une exécution pratique 
+Goxygen génère du code Backend en Go, le connecte aux composants Frontend, fournit un
+Dockerfile pour l'application et crée des fichiers Docker-compose pour une exécution pratique
 dans des environnements de développement et de production.
 
 <table>
     <thead>
     <tr align="center">
-        <td colspan=4><b>Supported Technologies</b></td>
+        <td colspan=4><b>Technoligies supportées</b></td>
     </tr>
     </thead>
     <tbody>
     <tr align="center">
-        <td align="center">Front End</td>
+        <td align="center">FrontEnd</td>
```

Comment:
I think it is better to stick to the "Front End" version to be consistent with the rest of the translations.

```suggestion
        <td align="center">Front End</td>
```